MathlibTest/MathlibTestExecutable.lean

```diff
@@ -0,0 +1,11 @@
+import Lean
+import Std
+import Qq
+import Batteries
+import Aesop
+import ProofWidgets
+import Mathlib
+import Plausible
+
+def main : IO Unit := do
+  IO.println "Verified that an executable importing all of Mathlib can be built and executed."
```

Comment:
A preliminary comment: if you thought about this in the meantime, consider this moot:

I understand where the message comes from... but my first reaction was "but wasn't this about a bug when importing all of aesop?". What do you think about clarifying the message, to mention e.g. mathlib and aesop? (Or is it guaranteed - and not just current behaviour - that mathlib imports all of aesop? In that case, things may be fine. But when we wouldn't need to put `import Aesop` in the test either.)

Reply:
New error message is clearer, thanks!